### DISCUSSION TITLE
DNS Problems with 4.2.0.

### DISCUSSION THREAD

**Discussion by Fred-DTV** [nextcloud/all-in-one#1828] [Questions] [answered]
Hi @szaimen 

I had issues with the mail server not working anymore so I decided to run the update you released today straight away.

Unfortunaltely now my containers won't start properly anymore:
1. First the mastercontainer wouldn't start at all without the --dns setting which I didn't need before.
2. Then I tried to set the IP to my local DNS which is running on the same system and it still wouldn't start
3. When I set the DNS to 1.1.1.1 the mastercontainer was starting again but the nextcloud container shows the following error: 

```
Checking connection to appstore
Curl didn't produce a 200 status, is appstore reachable?
```

I am a little afraid tbh to delete the nextcloud container. Would that delete my data and settings too?

**Comment by szaimen** [comment, marked as answer]
Hi, can you check if configuring your docker daemon to use your local dns makes it work? https://dockerlabs.collabnix.com/intermediate/networking/Configuring_DNS.html

**Reply by Fred-DTV** [reply]
Thanks to your link I was able to find the Synology related solution:

```
Since you are using DNS server on the synology then try the following

From the main Synology UI select DNS server
Click on the "Resolution" menu.
If you have "Limit source IP service" checked then you need to enter the docker network subnet here such as [172.17.0.0](https://172.17.0.0/) with subnet 255.255.0.0
```

This plus --dns="1.1.1.1" worked and I guess it would work with the local dns too


Thanks again so much for your help!
